.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+image: 
+  file: .gitpod.Dockerfile
+tasks:
+- init: |
+    cp .env.example .env
+  command: |
+    mkdir -p ~/.kube
+    mkdir -p ~/.config/gcloud
+    open .env
+    gcloud auth login
+    make install
```

Comment:
If you run `make install` at the end of this won't it try to just launch the installer with the example values? That is before this user has had the chance to modify to suit their account. Perhaps a `make help` so they see what the options are plus a "edit .env and run make install when you are ready"

That confused me for a minute before I realized I didn't do anything wrong.

Reply:
For me, I opened the workspace saw the Google oauth prompt right away, connected that, before I edited the env. Knee jerk reaction to being prompted for a login, "do it quickly or something will timeout". Then I wasn't sure how to restart. Had to dig into the gitpod to understand get to oh just re-run make install.

Reply:
Yes, I agree this can be further streamlined. Maybe the install should initially check for the required env vars (BILLING_ACCOUNT and DOMAIN) and fail early if they haven't been set.
That said I'm still not sure what to do about the DOMAIN at all 😁